src/components/Chat/Addons.tsx

```diff
@@ -43,26 +41,20 @@ const Addon = ({
     <button
       className={classNames(
         `flex items-center gap-2 rounded px-3 py-2 text-left`,
-        { 'bg-blue-500/20': isSelected },
+        { 'bg-accent-primary': isSelected },
         {
-          'bg-gray-100 hover:bg-gray-400 dark:bg-gray-700 hover:dark:bg-gray-600':
-            !isSelected,
+          'bg-layer-3 hover:bg-layer-4': !isSelected,
         },
       )}
       disabled={preselectedAddonsIds.includes(addonId)}
       onClick={() => {
         onChangeAddon(addonId);
       }}
     >
-      <ModelIcon
-        entity={addonsMap[addonId]}
-        entityId={addonId}
-        size={15}
-        inverted={!addonsMap[addonId]?.iconUrl && theme === 'dark'}
-      />
+      <ModelIcon entity={addonsMap[addonId]} entityId={addonId} size={15} />
```

Comment:
I suppose `entity` already includes `entityId` and we can decrease argument count